testsuite/forge/src/backend/k8s/chaos.rs

```diff
@@ -166,12 +178,92 @@ impl K8sSwarm {
         ))
     }
 
+    fn create_netem_template(&self, swarm_netem: &SwarmNetEm) -> Result<String> {
+        let mut network_chaos_specs = vec![];
+
+        for group_netem in &swarm_netem.group_netems {
+            let source_instance_labels = group_netem
+                .source_nodes
+                .iter()
+                .map(|node| {
+                    if let Some(v) = self.validator(*node) {
+                        v.name()
+                    } else {
+                        "invalid-node"
+                    }
+                })
+                .collect::<Vec<_>>()
+                .join(",");
+
+            let target_instance_labels = group_netem
+                .target_nodes
+                .iter()
+                .map(|node| {
+                    if let Some(v) = self.validator(*node) {
+                        v.name()
+                    } else {
+                        "invalid-node"
+                    }
+                })
+                .collect::<Vec<_>>()
+                .join(",");
+
+            network_chaos_specs.push(format!(
+                include_str!(NETEM_CHAOS_TEMPLATE!()),
+                name = &group_netem.name,
+                namespace = self.kube_namespace,
+                delay_latency_ms = group_netem.delay_latency_ms,
+                delay_jitter_ms = group_netem.delay_jitter_ms,
+                delay_correlation_percentage = group_netem.delay_correlation_percentage,
+                loss_percentage = group_netem.loss_percentage,
+                loss_correlation_percentage = group_netem.loss_correlation_percentage,
+                instance_labels = &source_instance_labels,
+                target_instance_labels = &target_instance_labels,
+                rate = group_netem.rate_in_mbps,
+            ));
+        }
+
+        Ok(network_chaos_specs.join("\n---\n"))
+    }
+
+    fn create_cpu_stress_template(&self, swarm_cpu_stress: &SwarmCpuStress) -> Result<String> {
```

Comment:
Having a comment here (or somewhere else relevant) explaining how the CPU stress work would be useful.